cities_by_population.rb

```diff
@@ -0,0 +1,25 @@
+$LOAD_PATH << '.'
+require 'city'
+
+module CitiesByPopulation 
+      
+  def sort_cities_by_population(cities)
+      file = File.open('Cities_By_Population.txt', 'w') do |file|
+      text_to_save = construct_city_by_population(cities)
+      file.puts text_to_save
+    end  
+  end
+    
+  def construct_city_by_population(cities)
+    text_to_save=""
+    cities.sort_by(&:population).reverse.group_by(&:population).each do |population, city|
```

Comment:
move this `cities.sort_by(&:population).reverse.group_by(&:population)` to a method, it is hard to understand that logic. There is something call the Law of Demeter, should read about it